auth_admin_passkey/__openerp__.py

```diff
@@ -1,28 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2016 GRAP
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 # -*- encoding: utf-8 -*-
-##############################################################################
-#
-#    Admin Passkey module for OpenERP
-#    Copyright (C) 2013-2014 GRAP (http://www.grap.coop)
-#    @author Sylvain LE GAL (https://twitter.com/legalsylvain)
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
 {
     'name': 'Authentification - Admin Passkey',
-    'version': '8.0.2.1.1',
+    'version': '9.0.1.0.0',
```

Comment:
I think the minors of the version number should be preserved (i.e. 9.0.2.1.1) since the feature set has not changed.